src/components/auth/SettingsModal.tsx

```diff
@@ -45,11 +46,15 @@ export const SettingsMenuItem = React.memo(function SettingsMenuItem({ finalRef
 
     const profile = useSelector((state: AppState) => ProfileSelectors.getProfile(state, userId ?? ""));
     const toskiPlayer = getPlayerName(profile ? profile.id : "");
+    const defaultMoxfieldLogo = "https://pbs.twimg.com/profile_images/1674989472839094273/p7a37K9W_400x400.jpg";
```

Comment:
move to outside of the component

Reply:
(constants should live outside the component. if you are putting them inside the component, they are getting recreated every time the component renders)